web/Services/TenantService.cs

```diff
@@ -44,10 +44,8 @@ public TenantService(BricksAndHeartsDbContext dbContext, IPostcodeService postco
                 FROM dbo.Tenant
                 WHERE Lon is not NULL and Lat is not NULL
                 ORDER BY distance
-                OFFSET {perPage * (page - 1)} ROWS
-                FETCH NEXT {perPage} ROWS ONLY
                 "
             );
-        return await tenants.ToListAsync();
+        return tenants.AsEnumerable();

```

Comment:
So we should be using ToList here. A List is ordered, whereas an Enumerable is not, this means that the ordering here is not guaranteed to remain. Its the same in the other services too :)